### ISSUE TITLE
Neomorphism Accessibility Concerns

### ISSUE THREAD

**Issue by crutchcorn** [CuriousNikhil/neumorphic-compose#3]
Hey @CuriousNikhil! First off, thanks for your public, exploratory work on implementing this relatively new online aesthetic. If nothing else, it's been exciting reading through the codebase as a form of reference for other UI patterns.

With that said, I’m curious how you see someone implementing this with accessibility in mind.  There are some [issues when it comes to the neuromorphic design patterns and accessibility](https://uxdesign.cc/lets-talk-neumorphism-and-accessibility-44a48a6ace72). Among the issues, there are:
 • Lack of proper contrast
 • Lack of distinct contrasting component states
 • Increased cognitive load in app usage

I’d love to see more focus on those concerns as you continue building this out! I think it would be a great way to further explore the viability of something like this. If you have any thoughts or questions I’d love to discuss them!

**Comment by crutchcorn** [comment]
I've updated the tone of the comment above (see also the previous title). I shouldn't have been as harsh as I was, and I apologize. I saw something I felt strongly about, hadn't slept in far too long, and should have waited until the morning to draft this/hit send. I hope we can have a positive discourse about accessibility - my tone won't dip back to that again. Apologies again

**Comment by CuriousNikhil** [comment]
Hi @crutchcorn, 
I know and I'm aware that there are issues regarding Accessibility when it comes to Neumorphism UI and why that is not being completely adopted piece of tech. I am glad that you have raised this issue. You don't have to apologise for it 😄. 
Of course, I would not recommend to use this experimental Neumorphism UI library in production. This is just an experimentation as I am learning and playing with jetpack compose. 
I would be happy to discuss related questions that I have with you and would like your inputs on this.

**Comment by crutchcorn** [comment]
Totally understandable! You're making great progress with it, as well! 😊

Do you think it might be reasonable to include a warning in the README highlighting the accessibility concerns?

**Comment by CuriousNikhil** [comment]
Yes. Will add warning in the README for the same.
Thanks.
